tools/src/visitor.ts

```diff
@@ -813,7 +813,7 @@ export default class Visitor {
 								}
 								const entry: EnumerationEntry = { name: declaration.name.getText(), value: value };
 								// Use the declaration statement since enum declaration only have one declaration.
-								this.fillDocProperties(variable.declarationList, entry);
+								this.fillDocProperties(variable.declarationList.parent, entry);
```

Comment:
Shouldn't the then be `this.fillDocProperties(variable, entry)` instead of going down one and up one.

We should also fix the comment above.

Reply:
Oh, ofc - I totally misread that the first time. Fixed!